NodeJS-server/controllers/news.controller.js

```diff
@@ -0,0 +1,129 @@
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+const ValidationError = require('../errors/ValidationError.component')
+
+function getArticleById(req, res, next) {
+  fs.readFile('./mockData.json', 'utf8', (err, data) => {
```

Comment:
Usually it's better to have a separate module with unified CRUD methods (get, find, create, update, remove) that can be reused. But in case with only one model your approach is also fine.
You can also try async / await here.